.github/workflows/pull-request-management.yml

```diff
@@ -780,3 +782,45 @@ jobs:
         # For most runs without an environment set here. The environment is mapped with the tox gh-action plugin.
         run: |
           tox -- pyavd-utils/tests
+
+  # --------------------------------------- #
+  # Build cross platform pyavd-utils wheels
+  # --------------------------------------- #
+  pyavd_utils_build_wheels:
+    name: Build pyavd-utils wheel on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    needs: [file-changes]
+    if: needs.file-changes.outputs.rust_src_or_ci_changed == 'true' || github.event_name == 'merge_group'
+    strategy:
+      fail-fast: false
+      matrix:
+        # macos-14 covers x64 and macos-latest covers arm
+        # note that at some point x64 won't be available anymore as per
+        # https://github.com/actions/runner-images/issues/13046
+        # the next update will be macos-15-intel
+        os:
+          - ubuntu-latest
+          - ubuntu-24.04-arm
+          - windows-latest
+          - macos-14
+          - macos-latest
+    steps:
+      - uses: actions/checkout@v5
+      - uses: actions/setup-python@v6
+        with:
+          python-version: "3.12"
+      - name: Install cibuildwheel
+        # See pyproject.toml for config
+        # TODO: could use the action instead but need approval
+        run: |
+          pip install --group build_wheel
+      - name: Build wheels
+        # Not using working-directory
+        # because manylinux uses a container and will not copy the
+        # parent Cargo.toml and we will be sad
```

Comment:
```suggestion
        # Not using working-directory because manylinux uses a container
        # and will not copy the parent Cargo.toml and we will be sad
```